### ISSUE TITLE
Enable online search by default, or at least move the "Internet" up

### ISSUE THREAD

**Issue by tobiasBora** [osmandapp/Osmand#8118]
Hello,

First thanks a lot for this great project. OsmAnd is great, but the most painfull feature is search for me, which is usually not very relevant and pretty slow (ok, it's a phone, I understand that it's slow). However, the online search is much faster, and usually more accurate (not perfect but still better). Would it be possible to add an option to easily enable online search by default (and why not let the user customize the search method/server, with [Photon](http://photon.komoot.de/) and/or [Nominatim](Nominatim)) ? 

The way I would see it is as follow:

1) First, change the "Internet" button in the search tab into 3 buttons: `[Local][Web][Both]`. Pressing the `Local` button would run a local search, pressing the `Web` button would run an online search (it's the current behaviour of the button), and the last option would run both a local and web search, and display them together (not sure how to mix them, maybe one line local/one line web/one line local/... or eventually based on some thiner popularity/relevance properties if someone is motivated to implement it).
2) Display this button even if no search has been done (for now you need first to do a local search to do an internet search... So you can wait like 5s before starting your internet query)
3) If a search has been done, this button should be placed in a visible place (for now the button appears after the long list of occurences so you need to scroll down for a few more secondes to reach the button). I'd say the best place is to put it right above the keyboard when the user is typing, and stuck it to the bottom of the screen otherwise, more or less like Firefox when it displays the search engines choices. But it could also be possible to put right after the input line, or at the end of the input line (not nice for small screens maybe), or eventually on the same line as the `See on map` button to save space.
4) Add a way to let the user customize in the Settings which is the default search (Local/Web/Both).

What do you think?

Thanks again for the great job!

**Comment by scaidermern** [comment]
Related to #7716.

**Comment by sonora** [comment]
Tactically, I am not sure this is only "Nice to Have" ...

We have long been faced with reproducible criticism regarding OsmAnd's offline search capability and performance. And while I am well aware of the technical limitations and challenges, I think offering a well visible "Online search" capability may well be the workaround we need here in order to create leeway to solve the offline search issues - even if 'offline' is and remains our main focus.

**Comment by scaidermern** [comment]
I fully agree with @sonora. Currently offline search is one of OsmAnd's major drawbacks. Online search should be made more accessible and needs to be improved (#7123).
